core/src/components/searchbar/searchbar.tsx

```diff
@@ -176,6 +182,10 @@ export class Searchbar implements ComponentInterface {
     });
   }
 
+  componentWillLoad() {
```

Comment:
```suggestion
  connectedCallback() {
```

so if moved to different places, we can still re-fire